packages/query/src/stacks/mempool/mempool.query.ts

```diff
@@ -0,0 +1,29 @@
+import { MempoolTransactionListResponse } from '@stacks/stacks-blockchain-api-types';
+import { useQuery } from '@tanstack/react-query';
+
+import { useHiroApiRateLimiter } from '../../hiro-rate-limiter';
+import { useStacksClient } from '../stacks-client';
+
+export interface UseAccountMempoolQueryArgs {
+  address: string;
+  onSuccess?(data: MempoolTransactionListResponse): void;
```

Comment:
We'll need to not rely on this in v5

Reply:
@kyranjamie nice article on why those callbacks were removed: https://tkdodo.eu/blog/breaking-react-querys-api-on-purpose